portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -1564,4 +1564,80 @@ definition {
 			title = "WC Title");
 	}
 
+	@description = "When the user clicks on My Sites or Asset Libraries tabs within the modal, the list of Sites or Asset Libraries to which the user has NO access is NOT displayed"
+    @priority = "5"
+    test SitePermissions {
+        property portal.acceptance = "true";
+
```

Comment:
Please add tasks to divide the test and make it more readable. See: https://issues.liferay.com/browse/LPS-130691